utils/coreutils/tableutils.go

```diff
@@ -370,12 +370,18 @@ type embeddedTableCell struct {
 // ┌─────────────────────────────────────────┐
 // │ An example of a message in a nice frame │
 // └─────────────────────────────────────────┘
-func PrintMessage(message string) {
+func PrintMessageInsideFrame(message, marginLeft string) {
```

Comment:
Let's add a comment above this function with an example of how the `marginLeft` value can change the frame view.